faq/databases-for-postgresql-and-mysql.mdx

```diff
@@ -142,28 +142,28 @@ The following `pg_extensions` are available for Scaleway Database:
 
 
 If the automatic backup feature is enabled, new backups will be created according to your backup schedule, which can be set on the "Instance overview” tab. Your Instance is available during backup, but there might be a performance impact and some actions may not be available.
-Automatic backups are enabled by default. It is possible to disable them at any time in the backup settings of your Instance. 
+Automatic backups are enabled by default. It is possible to disable them at any time in the backup settings of your Instance.
 It is also possible to launch manual backups by clicking on the **Create a backup** button from the backups tab.
 
 ## Are my active and standby database nodes in a high-availability cluster hosted in the same datacenter?
 
-In a high-availability cluster, active and hot standby nodes are indeed located in the same datacenter but in two separated racks.  
+In a high-availability cluster, active and hot standby nodes are indeed located in the same datacenter but in two separated racks.
 The idea is to offer the best performance to ours users by reducing latency between active and hot standby nodes, as we use a sync replication process between the nodes.
 
 ## What is the Private Networks feature for Database Instances?
 
-[Private Networks](/network/public-gateways/quickstart/) allows you to configure your own Layer-2 network with IPv4 addresses. You can therefore enhance the security of your system architecture by isolating it from the Internet. 
+[Private Networks](/network/public-gateways/quickstart/) allows you to configure your own Layer-2 network with IPv4 addresses. You can therefore enhance the security of your system architecture by isolating it from the Internet.
 
-This feature introduces a significant change in the architecture of Scaleway's Databases: when using Private Networks, your application connects directly to your Database nodes, without going through a Load Balancer. 
+This feature introduces a significant change in the architecture of Scaleway's Databases: when using Private Networks, your application connects directly to your Database nodes, without going through a Load Balancer.
 This improves performance by reducing the latency between your nodes. It also increases the security of your databases, since Instances in your Private Network can directly to your Database Instance, without passing through the public Internet.
```

Comment:
```suggestion
This improves performance by reducing the latency between your nodes. It also increases the security of your databases, since Instances in your Private Network can connect directly to your Database Instance, without passing through the public Internet.
```